ytcomment.py

```diff
@@ -0,0 +1,91 @@
+from helpers.analysis import calc_sentiment
+class YTComment:
+    def __init__(self, data):
+        self.value = ""
+        self.likes = 0
+        self.sentiment = 0
+        #
+        if "snippet" in data:
+            if "topLevelComment" in data["snippet"]:
+                if "snippet" in data["snippet"]["topLevelComment"]:
+                    real_data = data["snippet"]["topLevelComment"]["snippet"]
+                    if "textDisplay" in real_data:
+                        self.value = real_data["textDisplay"]
+                    if "likeCount" in real_data:
+                        self.likes = real_data["likeCount"]
+
+        self.sentiment = calc_sentiment(self.value)
```

Comment:
What if value is ""?

Reply:
It returns 0, and will therefore not have an impact on the data average/quartiles since we are using median instead of mean.

Reply:
Do tertiary or an if statement to assure that self.value is not just an empty string. It is expensive to be making a call like this for it to just return a 0.

Reply:
Gotcha